fixed grammar in signed-exchanges-caveats.mdx (#16465)
cloudflare/cloudflare-docs: src/content/docs/speed/optimization/other/signed-exchanges/signed-exchanges-caveats.mdx

* `Strict-Transport-Security`
* `WWW-Authenticate`

> Signed exchanges will not be created if the response headers contain a `cache-control` header whose value is one of:

Reviewer: ```suggestion
Signed exchanges will not be created if the response headers contain a `cache-control` header with one of the following values:
```
